tests/components/searchDrug/index.test.jsx

```diff
@@ -1,106 +0,0 @@
-import React from 'react';
-import _ from 'lodash';
-
-import ConnectedSearchDrug, {
-  SearchDrug
-} from '../../../app/js/components/searchDrug';
-
-
-let props;
-let mountedComponent;
-let clock;
-
-jest.useFakeTimers();
-
-const getComponent = () => {
-  if (!mountedComponent) {
-    mountedComponent = shallow(<SearchDrug {...props} />);
-  }
-  return mountedComponent;
-};
-
-describe('Component: SearchDrug: Container', () => {
-  beforeEach(() => {
-    props = {
-      drugSearch: {
-        drugs: []
-      },
-      searchDrug: jest.fn(),
-      onChange: jest.fn(),
-      onOptionSelected: jest.fn(),
-      selectDrug: jest.fn(),
-      onChange: jest.fn(),
-      onSelectDrug: jest.fn(),
-    };
-    mountedComponent = undefined;
-  });
-
-  it('should render properly', () => {
-    const component = getComponent();
-    expect(component).toMatchSnapshot();
-  });
-
-  describe('Interactions', () => {
-    beforeEach(function () {
-      clock = sinon.useFakeTimers();
-    });
-
-    afterEach(function () {
-      clock.restore();
-    });
-
-    it('should call searchDrug() with debounce feature', () => {
-      const searchDrugSpy = sinon.spy(props, 'searchDrug');
-      sinon.spy(getComponent().instance(), 'searchDrug');
-      getComponent().instance().searchDrug('para');
-      expect(getComponent().instance().searchDrug.calledOnce)
-        .toEqual(true);
-
-      clock.tick(500);
-      expect(searchDrugSpy).toHaveBeenCalled();
-    });
-
-    it('should not call searchDrug()', () => {
-      const searchDrugSpy = sinon.spy(props, 'searchDrug');
-      sinon.spy(getComponent().instance(), 'searchDrug');
-      getComponent().instance().searchDrug('  ');
-
-      clock.tick(500);
-      expect(searchDrugSpy).not.toHaveBeenCalled();
-    });
-
-    it('should call onChange()', () => {
-      const event = {
-        target: {
-          value: "Value"
-        },
```

Comment:
Was this also part of the bug fix?

Reply:
The file was causing all our tests to fail, so had to delete it in other to get them passing, and the fixing of the file will be worked on as a separate ticket